### PR TITLE
fix: block number isn't formatted

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/history/events/HistoryEventsList.vue [hunk 1]
```diff
@@ -194,7 +205,11 @@ const { mdAndUp } = useDisplay();
                 <template #item.description="{ item }">
                   <VLazy>
                     <HistoryEventNote
-                      v-bind="item"
+                      v-bind="{
+                        ...headerItems,
+                        ...item
+                      }"
+                      notes="Receive 0.021208049430266542 ETH from 0x1f9090aaE28b8a3dCeaDf281B0F12828e676c326 as mev reward for block 17607841"
```
Comment (kelsos) [inline frontend/app/src/components/history/events/HistoryEventsList.vue:212]: This certainly should not be part of the PR 😄

File: frontend/app/src/components/history/events/HistoryEventsList.vue [hunk 2]
```diff
@@ -128,6 +128,17 @@ watch(
...
+  const items: Record<string, any> = {};
+  const blockEvent = get(isEthBlockEventRef(eventGroupHeader));
+
+  if (blockEvent) {
+    items.blockNumber = blockEvent.blockNumber.toString();
+  }
+
+  return items;
+});
+
```
Comment (kelsos) [inline frontend/app/src/components/history/events/HistoryEventsList.vue:141]: Are we going to add more stuff in the items? it seems a bit redundant.

you either return `{ blockNumber: blockEvent.blockNumber.toString() }` or `{}`

Comment (lukicenturi) [reply frontend/app/src/components/history/events/HistoryEventsList.vue:141]: For now yes it's only blockNumber, just prepared if in the future we need more fields.
Should I just do it directly only for blockNumber for now?
```
blockNumber: eventGroupHeader.blockNumber?.toString() || item.blockNumber
```

Comment (kelsos) [reply frontend/app/src/components/history/events/HistoryEventsList.vue:141]: No need to complicate it, just keep it simple and we can adjust it in the future if the need arises :)

Comment (lukicenturi) [reply frontend/app/src/components/history/events/HistoryEventsList.vue]: updated
